internal/core/core_test.go

```diff
@@ -45,7 +45,7 @@ type fakeFileClient struct {
 	statResponses []any
 }
 
-func (f *fakeFileClient) Stat(ctx context.Context, in *fpb.StatRequest, opts ...grpc.CallOption) (*fpb.StatResponse, error) {
+func (f *fakeFileClient) Stat(_ context.Context, _ *fpb.StatRequest, _ ...grpc.CallOption) (*fpb.StatResponse, error) {
```

Comment:
optional: since none of the names are provided, you can omit the names entirely:
`func (f *fakeFileClient) Stat(context.Context, *fpb.StatRequest, ...grpc.CallOption) (*fpb.StatResponse, error) {`